locales/en/create-kafka-instance-with-sizes.json

```diff
@@ -0,0 +1,109 @@
+    "system_unavailable_message": "$t(common:unexpected_error)",
+    "system_unavailable_title": "$t(common:something_went_wrong)",
+    "trial_available_message": "The trial has a duration of {{time}} hours. If you are interested in using this service in production, go to our <0>Streams for Apache Kafka Overview</0> page to see what options are available during creation and learn how to contact us.",
+    "trial_available_title": "You can create a trial instance to evaluate this service.",
+    "trial_unavailable_message": "All available trial instances are currently in use. Try again later.",
+    "trial_unavailable_title": "$t(common:kafka) instances are unavailable for creation",
+    "trial_used_message": "You can deploy 1 instance at a time. To deploy a new instance, delete your existing one first. If you are interested in using this service in production, go to the <0>Streams for Apache Kafka Overview</0> to see what options are available during creation and learn how to contact us.",
+    "trial_used_title": "You already have an instance running"
+  },
+  "no_regions_available": "No regions available",
+  "partitions_value": "up to {{value}}",
+  "quick_start_guide_message": "Need help getting started? Follow our quick start guide.",
+  "quick_start_guide_message_exp": "Follow our quick start guide.",
+  "quick_start_guide_title": "Need help getting started?",
+  "size_field_aria": "More info for Size field",
+  "size_help_content": "Size of a Kafka instance is based on streaming units. The number of streaming units defines the capacity, resources, and limits of an instance. An instance with a larger size can handle higher loads and process larger amounts of events, has more storage, and can handle more clients and connections.",
+  "standard_kafka_size_description": "Learn how to add streaming units to your account",
+  "standard_kafka_streaming_unit": "Your organization has {{count}} streaming unit remaining",
+  "standard_kafka_streaming_unit_zero": "Your organization has no streaming units remaining",
+  "standard_kafka_streaming_unit_one": "Your organization has {{count}} streaming unit remaining",
+  "standard_kafka_streaming_unit_other": "Your organization has {{count}} streaming units remaining",
+  "select_cloud_provider": "Select cloud provider",
+  "select_region": "Select region",
+  "sizes_missing": "Size options display when a cloud provider and region are selected",
+  "statuses": {
+    "creating": "Creating",
+    "degraded": "Degraded",
+    "deleting": "Deleting",
+    "ready": "Ready"
+  },
+  "storage_value": "up to {{value}} GiB",
+  "stream_unit_value": "{{size}} streaming units",
+  "streaming_unit": "streaming units",
+  "trial_kafka_size_description": "With a subscription, you can select a size for your instance. Changing the size allows you to increase limits such as ingress, egress, and storage. The limits that you get with a trial instance are listed under Details.",
```

Comment:
PM requested to make this shorter. All of the contents need a full review from our content designer, but for now, let's change this to the following:
> With a subscription, you can change the capacity of your instance by selecting a size. The capacity that you get with a trial instance is listed under Details.